sql/src/main/java/io/crate/operation/merge/RamAccountingPageIterator.java

```diff
@@ -1,78 +1,35 @@
 package io.crate.operation.merge;
 
 import com.google.common.annotations.VisibleForTesting;
-import io.crate.breaker.RamAccountingContext;
 import io.crate.breaker.RowAccounting;
 import io.crate.data.Row;
-import io.crate.operation.projectors.sorting.OrderingByPosition;
-import io.crate.planner.PositionalOrderBy;
-import io.crate.planner.node.dql.MergePhase;
-import org.elasticsearch.common.breaker.CircuitBreaker;
+import org.elasticsearch.common.Nullable;
 
 import java.util.function.Consumer;
 
 /**
  * Wraps a PagingIterator and uses {@link io.crate.breaker.RamAccountingContext} to apply the circuit breaking logic
  */
-public class RamAccountingPageIterator implements PagingIterator<Integer, Row> {
+class RamAccountingPageIterator implements PagingIterator<Integer, Row> {
 
     @VisibleForTesting
     final PagingIterator<Integer, Row> delegatePagingIterator;
+    @Nullable
     private final RowAccounting rowAccounting;
 
-    private RamAccountingPageIterator(PagingIterator<Integer, Row> delegatePagingIterator,
-                                      RowAccounting rowAccounting) {
+    RamAccountingPageIterator(PagingIterator<Integer, Row> delegatePagingIterator,
+                                     @Nullable RowAccounting rowAccounting) {
```

Comment:
why is this now nullable? This doesn't make sense to be used without a RowAccounting instance?